app/src/main/java/ca/vijayan/flyweb/BrowseActivity.java

```diff
@@ -50,6 +54,18 @@ protected void onCreate(Bundle savedInstanceState) {
         settings.setBuiltInZoomControls(true);
         settings.setDisplayZoomControls(false);
         mWebView.setWebViewClient(new WebViewClient() {
+            @Override
+            public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest req) {
+                Uri uri = req.getUrl();
```

Comment:
I remember needing to handle the case where uri.getPort() == 0 (which defaults to using 80).